zad6/app/controllers/products_controller.rb

```diff
@@ -0,0 +1,76 @@
+class ProductsController < ApplicationController
+  before_action :set_product, only: [:show, :edit, :update, :destroy]
+  before_action :authenticate_customer!, except: [:index, :show]
+
+  # GET /products
+  # GET /products.json
+  def index
+    # Why products won't order?
```

Comment:
Hmmm, u mnie wyglądają na posortowane po cenie rosnąco, dokładnie tak jak tutaj jest napisane 🤔 
![screen shot 2018-12-11 at 13 03 42](https://user-images.githubusercontent.com/18199403/49799472-4b23ac80-fd45-11e8-99a9-eb5d810c9177.png)

Reply:
Byc może wystarczyło po prostu zrestartować serwer, a się przyzwyczaiłem, że nie ma takiej potrzeby i zapominam o tym, a zauważyłem, że niektóre tego typu błędy to potrafi rozwiązać